SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -43,6 +43,13 @@ USpatialGDKSettings::USpatialGDKSettings(const FObjectInitializer& ObjectInitial
 	, WorkerLogLevel(ESettingsWorkerLogVerbosity::Warning)
 	, SpatialDebuggerClassPath(TEXT("/SpatialGDK/SpatialDebugger/BP_SpatialDebugger.BP_SpatialDebugger_C"))
 	, bEnableUnrealLoadBalancer(false)
+	// TODO - UNR 2514 - These defaults are not necessarily optimal - readdress when we have better data
```

Comment:
Where do these values come from? Ideally, they should be equal to the ones we had by default before this PR, but I can't find these.

Reply:
From Worker SDK defaults - which aren't exposed.